gitter.py

```diff
@@ -190,7 +193,12 @@ def leave(self, reason=None):
 
     @property
     def topic(self):
-        return "TODO"  # TODO
+        json = self._backend.readAPIRequest('rooms', {"q": self.uri}).json()
+        for element in json:
+            if element.uri == self.uri:
+                return element.topic
+            else:
+                raise RoomNotFoundError
```

Comment:
I don't believe it's good practice to raise exceptions without a message. I would suggest the following: 

```python
raise RoomNotFoundError("Cannot find the room '{}'".format(self.name))
```